warehouse/ingest-scripts/src/main/resources/bin/system/stop-metrics-ingest.sh

```diff
@@ -9,8 +9,12 @@ else
 fi
 THIS_DIR="${THIS_SCRIPT%/*}"
 cd $THIS_DIR
+
 #stop scripts do not require force despite lock files
 . ../ingest/ingest-env.sh -force
+. ../util/logging_pdsh.sh
```

Comment:
logging_pdsh sources ingest-env, however without the -force arg. I'm wondering if there's a potential problem here. Maybe it's better not to source ingest-env in logging_pdsh, and instead require that anyone who sources it defines the things it needs (usually by having sourced ingest-env first).

Reply:
Sounds good. It looks like only PDSH_LOG_DIR is assumed to be in the environment anyway.